dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/WorkflowE2ETest.java

```diff
@@ -162,56 +162,56 @@ void testCreateSubWorkflow() {
-                .goToTab(WorkflowDefinitionTab.class)
-                .run(workflow)
-                .submit();
-
-        await().untilAsserted(() -> {
-            browser.navigate().refresh();
-
-            final Row row = projectPage
-                    .goToTab(WorkflowInstanceTab.class)
-                    .instances()
-                    .iterator()
-                    .next();
-
-            assertThat(row.isSuccess()).isTrue();
-            assertThat(row.executionTime()).isEqualTo(1);
-        });
-        // Test rerun
-        projectPage
-                .goToTab(WorkflowInstanceTab.class)
-                .instances()
-                .stream()
-                .filter(it -> it.rerunButton().isDisplayed())
-                .iterator()
-                .next()
-                .rerun();
-
-        await().untilAsserted(() -> {
-            browser.navigate().refresh();
-
-            final Row row = projectPage
-                    .goToTab(WorkflowInstanceTab.class)
-                    .instances()
-                    .iterator()
-                    .next();
-
-            assertThat(row.isSuccess()).isTrue();
-            assertThat(row.executionTime()).isEqualTo(2);
-        });
-    }
+//    @Test
+//    @Order(30)
+//    void testRunWorkflow() {
+//        final String workflow = "test-workflow-1";
+//        final ProjectDetailPage projectPage =
+//                new ProjectPage(browser)
+//                        .goToNav(ProjectPage.class)
+//                        .goTo(project);
+//
+//        projectPage
+//                .goToTab(WorkflowInstanceTab.class)
+//                .deleteAll();
+//        projectPage
+//                .goToTab(WorkflowDefinitionTab.class)
+//                .run(workflow)
+//                .submit();
+//
+//        await().untilAsserted(() -> {
+//            browser.navigate().refresh();
+//
+//            final Row row = projectPage
+//                    .goToTab(WorkflowInstanceTab.class)
+//                    .instances()
+//                    .iterator()
+//                    .next();
+//
+//            assertThat(row.isSuccess()).isTrue();
+//            assertThat(row.executionTime()).isEqualTo(1);
+//        });
+//        // Test rerun
+//        projectPage
+//                .goToTab(WorkflowInstanceTab.class)
+//                .instances()
+//                .stream()
+//                .filter(it -> it.rerunButton().isDisplayed())
+//                .iterator()
+//                .next()
+//                .rerun();
+//
+//        await().untilAsserted(() -> {
+//            browser.navigate().refresh();
+//
+//            final Row row = projectPage
+//                    .goToTab(WorkflowInstanceTab.class)
+//                    .instances()
+//                    .iterator()
+//                    .next();
+//
+//            assertThat(row.isSuccess()).isTrue();
+//            assertThat(row.executionTime()).isEqualTo(2);
+//        });
+//    }
```

Comment:
Because the input parameters of runworkflow are modified, the E2E is shielded for the time being. After the front-end code is merged, it is opened again.

Reply:
Currently our E2E test does not cover the timing module and the complement module. I think we should not change it.